DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitSendBinding.cs

```diff
@@ -57,25 +55,16 @@ public List<ISendFilter> GetSendFilters()
     return new List<ISendFilter> { new RevitSelectionFilter() { IsDefault = true } };
   }
 
-  public async Task Send(string modelCardId)
-  {
-    await SpeckleTopLevelExceptionHandler
```

Comment:
It looks like `SpeckleTopLevelExceptionHandler` is no longer used. Shall we delete the class also

Reply:
done!